src_base/xeve_port.h

```diff
@@ -141,9 +141,9 @@ void xeve_trace_line(char * pre);
 #define xeve_assert_gv(x,r,v,g) \
     {if(!(x)){assert(x); (r)=(v); goto g;}}
 
-
-
+#ifdef __SSE4.1__
```

Comment:
@Jamaika1 This seems not related to the main goal of this PR (in touch with the PR title). What about separating this commit as a new PR?

Reply:
To compile the codec I had to disable all functions above SSE2. Does the codec work without SIMD?

Reply:
@Jamaika1 Thank you for rectifying your commit. We will check out your PR and get back to you soon.